go/vt/vttablet/tabletserver/state_manager.go

```diff
@@ -330,6 +335,14 @@ func (sm *stateManager) CheckMySQL() {
 	}()
 }
 
+// isCheckMySQLRunning returns 1 if CheckMySQL function is in progress
+func (sm *stateManager) isCheckMySQLRunning() int64 {
```

Comment:
why int64 instead of bool?

Reply:
We return int64 because the gauge is of that type. So return 1 when the value is true, 0 otherwise.